public/admin.js

```diff
@@ -0,0 +1,21 @@
+const socket = io();
+
+const connectionCount = document.getElementById('connection-count');
+const statusMessage = document.getElementById('status-message');
+const choices = document.getElementById('survey-choices')
+const countedVotes = document.getElementById('results');
+const surveyId = document.getElementById('survey-id').innerText;
+
+socket.on('voteCount-' + surveyId.trim(), function (votes) {
+  if (countedVotes.children.length > 0) {
```

Comment:
What does this conditional do? It would probably be better if it was refactored out and had a name. It looks like you're just emptying out the element—so why not just do that?